lib/features/login/data/network/interceptors/authorization_interceptors.dart

```diff
@@ -106,8 +105,6 @@ class AuthorizationInterceptors extends QueuedInterceptorsWrapper {
           final uploadExtra = extraInRequest[FileUploader.uploadAttachmentExtraKey];
 
           requestOptions.headers[HttpHeaders.authorizationHeader] = _getTokenAsBearerHeader(_token!.token);
-          requestOptions.headers[HttpHeaders.contentTypeHeader] = uploadExtra[FileUploader.typeExtraKey];
-          requestOptions.headers[HttpHeaders.contentLengthHeader] = uploadExtra[FileUploader.sizeExtraKey];
```

Comment:
why we remove it?

Reply:
It is not necessary anymore, because we will reuse the previous request header.